examples/src/bin/basic_subclass.rs

```diff
@@ -160,7 +162,7 @@ mod imp_app {
         // application is launched by the desktop environment and
         // aksed to present itself.
         fn activate(&self, app: &gio::Application) {
```

Comment:
This would also become

```suggestion
        fn activate(&self, app: &SimpleApplication) {
```

once everything is updated?

Reply:
Right. The next step is to update the `*Impl` traits in gtk/etc to be defined similarly to how `ObjectExt` is handled here.